### PR TITLE
fix(sanitize): prefix ids from user generated html

### LINKED ISSUE #356: treat ids in html as we would other user generated ids
e.g.

```
<div id="add-billing-info">Foo.</div>
```

generates 
```
<div id="add-billing-info">Foo.</div>
```

when it should be

```
<div id="user-content-add-billing-info">Foo.</div>
```

### PR DESCRIPTION
**Pull request by ashleygwilliams**
fixes #356 
this will be a 9.0.3 release that we should do before tomorrow

### CODE REVIEW

File: lib/sanitize.js [hunk 1]
```diff
@@ -111,3 +123,12 @@ function sanitizeCellStyle (tagName, attribs) {
   var result = alignmentRegEx.exec(attribs.style || '')
   return result ? cell(result[1]) : cell()
 }
+
+function prefixHTMLids (tagName, attribs) {
+  if (attribs.id && attribs.id.replace('user-content-', '').length > 0) {
```
Comment (ChALkeR) [inline lib/sanitize.js]: What is the second check supposed to do here? It looks equivalent to `attribs.id !== 'user-content-'` atm.

Comment (ashleygwilliams) [inline lib/sanitize.js]: correct- i'm fixing it rn

Comment (ashleygwilliams) [inline lib/sanitize.js]: basically the idea is that we need to check that we didn't already namespace the user content id during the marky parse

**Comment by ashleygwilliams** [comment]
oh heads up standard breaks because of a thing we fixed for 10.0.0 otherwise it's good to go, so don't be worried by the breaking test, it's just (the latest version of)standard and i dont think it's worth trying to fix for a patch release

**Comment by ashleygwilliams** [comment]
whew actually just cherry-picked the standard fixes so travis should be happy now. good call @revin

**Comment by revin** [comment]
This doesn't need to get merged back into anything, just published from here. Closing, and thanks @ashleygwilliams :)

**Pull request closed without merging**